### PR TITLE
added shuffled array

### PR DESCRIPTION
**Pull request by manu-prakash-choudhary**
### Describe your change:

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: other/shuffled_array.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            any = any[:] + str(len(self.arr) // 2)
+        if int(any[-1]) < num:
+            return int(any[-1])
+        return self.prng(num)
+
+    def reset(self) -> list:
+        # it will return the original given array
+        """
+        >>> Solution([2,3,4,56]).reset()
+        [2, 3, 4, 56]
```
Comment (cclauss) [inline other/shuffled_array.py:49]: Let's test an unsorted sequence that includes negative ints, zero, and floating point numbers.
```suggestion
        >>> Solution([-7, 0, 4, -56.7]).reset()
        [-7, 0, 4, -56.7]
```

File: other/shuffled_array.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        >>> Solution([2]).shuffle()
+        [2]
+        """
+        temp = self.arr.copy()
+        for i in range(len(self.arr)):
+            a = self.prng(len(self.arr))
+            temp[a], temp[i] = temp[i], temp[a]
+        return temp
+
+
```
Comment (cclauss) [inline other/shuffled_array.py:66]: See CONTRIBUTING.md.  Your need `if __name__ == "__main__"`

```

**Comment by manu-prakash-choudhary** [comment]
@cclauss can you please review now

**Comment by manu-prakash-choudhary** [comment]
@cclauss

**Pull request closed without merging**